<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addUsers</class>
 <widget class="QDialog" name="addUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:century gothic;
color:white;
}
QLabel{
font-size:16px;
}
#head{
border-bottom-left-radius: 20px;
border-bottom-right-radius: 20px;
background: qlineargradient(spread:pad, x0:1, y1:0, x2:1, y2:1,stop:0 #6284FF, stop:1 #FF0000);
font-size:24px;
font-weight:bold;
}
QLineEdit{
font-size:16px;
font-weight:bold;
color:black;
background:white;
}
#birthdayValidate, #pExpiryValidate, #cellValidate, #homeValidate{
font-size:12px;
color:red;
background:transparent;
}
#addUsers,#frame{
background:#282828;
}
#packageDescriptionShow{
color:black;
font-size:15px;
background-color:white;
}
QComboBox{
font-size:16px;
background:#3e3f40;
}

QComboBox QAbstractItemView {
font-size:16px;
background-color:#3e3f40;
color:white;
}
QPushButton {
font-size:15px;
font-weight:bold;
border:none;
border-radius:5px;
background: #ff0303;
}

QPushButton:hover {
background:#49ebff;
color:black;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>678</width>
     <height>628</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>676</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD CUSTOMER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>641</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="middleNameEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lastNameEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="firstNameEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="middleNameLabel">
       <property name="text">
        <string>Middle Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>641</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="passportExpLabel">
       <property name="text">
        <string>Passport Expiry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passportEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="passportExpEdit">
       <property name="placeholderText">
        <string>mm/dd/yy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="birthdayLabel">
       <property name="text">
        <string>Birthday </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="birthdayEdit">
       <property name="placeholderText">
        <string>mm/dd/yy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="passportLabel">
       <property name="text">
        <string>Passport Number</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>641</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="postalLabel">
       <property name="text">
        <string>Postal Code/Zip Code</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cityEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="postalEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Street Address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="addressEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>641</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="2">
      <widget class="QLabel" name="cellPhoneLabel">
       <property name="text">
        <string>Cell Phone</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="homePhoneEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>XXXX-XXXXXXX</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="cellPhoneEdit">
       <property name="placeholderText">
        <string>XXXX-XXXXXXX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="countryLabel">
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="countryEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="homePhoneLabel">
       <property name="text">
        <string>Home Phone</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="packageDescriptionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>448</y>
      <width>637</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Package Description</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>641</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="1" column="0">
      <widget class="QLineEdit" name="numOfTravEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="numOftravLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string>Number Of Partcipants</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>313</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="ratingsLabel">
       <property name="text">
        <string>Ratings Given By Customer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ratingsSelect">
       <property name="maxCount">
        <number>5</number>
       </property>
       <property name="placeholderText">
        <string>1</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="packageDescriptionShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>593</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>593</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="packageSelect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>415</y>
      <width>511</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>selectPackage</string>
    </property>
   </widget>
   <widget class="QLabel" name="packageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>415</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Package</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="pExpiryValidate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>195</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>follow the format &quot;mm/dd/yy&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="homeValidate">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>340</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>follow the format &quot;xxxx-xxxxxxx&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthdayValidate">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>195</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>follow the format &quot;mm/dd/yy&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cellValidate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>follow the format &quot;xxxx-xxxxxxx&quot;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
